google-cloud-bigtable/acceptance/bigtable/table_test.rb

```diff
@@ -120,4 +120,33 @@
       [true, false].must_include result
     end
   end
+
+  describe "IAM policies and permissions" do
+    let(:service_account) { bigtable.service.credentials.client.issuer }
+    let(:table) { bigtable_read_table }
+
+    it "test permissions" do
```

Comment:
nit: s/test/tests/

Reply:
slippery slope with grammar corrections... this is google-cloud-bigtable.

Reply:
Fixed in existing `Instance` method as well as in new `Table` method.